### PR TITLE
update info

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9837

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -335,6 +335,9 @@ Sign-in a workstation with access equivalent to a _domain user_.
 > [!IMPORTANT]
 > For more details about the actual experience after everything has been configured, please see [Windows Hello for Business and Authentication](./hello-how-it-works-authentication.md).
 
+    > [!NOTE]
+    > For access issues in the context of VPN make sure to check the resolution and workaround described in  [this article](https://docs.microsoft.com/en-us/troubleshoot/windows-client/group-policy/group-membership-changes-not-updating-over-some-vpn-connections#workarounds).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:339]:
```suggestion
> [!NOTE]
> For access issues in the context of VPN, make sure to check the resolution and workaround described in  [this article](https://docs.microsoft.com/en-us/troubleshoot/windows-client/group-policy/group-membership-changes-not-updating-over-some-vpn-connections#workarounds).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @VLG17, kindly commit the suggested change if you agree with it. Many thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 2]
```diff
@@ -335,6 +335,9 @@ Sign-in a workstation with access equivalent to a _domain user_.
 > [!IMPORTANT]
 > For more details about the actual experience after everything has been configured, please see [Windows Hello for Business and Authentication](./hello-how-it-works-authentication.md).
 
+> [!NOTE]
+> For access issues in the context of VPN, make sure to check the resolution and workaround described in  [this article](https://docs.microsoft.com/en-us/troubleshoot/windows-client/group-policy/group-membership-changes-not-updating-over-some-vpn-connections#workarounds).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:339]:
```suggestion
> For access issues in the context of VPN, make sure to check the resolution and workaround described in [Workaround for user security context and access control](/troubleshoot/windows-client/group-policy/group-membership-changes-not-updating-over-some-vpn-connections#workarounds).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@VLG17 - you were a bit too quick for me and committed my changes before I completed the edit. Could you please also commit this suggested change (note that we have to use relative links)? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by mapalko — APPROVED** [review]

**Comment by e0i** [comment]
@tiburd This PR has been approved by the document author and can be merged. Thanks.

**Pull request merged**